src/environments/environment.ts

```diff
@@ -4,8 +4,7 @@
 
 export const environment = {
   production: false,
-  // note: this is a temporary value to satisfy function signatures; it does not suggest that a server should be running on port 3000
-  url: 'http://localhost:3000/',
+  url: 'http://avx:3000/dbb/oset/api',
```

Comment:
Can you revert this change? We shouldn't have anything with AV in it.

Reply:
One caveat: this change _does_ make sense to keep in the forked code for the demo; however, it shouldn't be merged to the `main` branch of the repo.

Reply:
I will revert it